draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -758,12 +779,224 @@ Both the `vct` claim in the SD-JWT VC and the various URIs in the Type Metadata
+       key in the currently selected element(s). If any of the currently
+       selected element(s) is not an object, abort processing and return an
+       error. If the key does not exist in any element currently selected,
+       remove that element from the selection.
+    2. If the `path` component is `null`, select all elements of the currently
+       selected array(s). If any of the currently selected element(s) is not an
+       array, abort processing and return an error.
+    3. If the `path` component is a non-negative integer, select the element at
+       the respective index in the currently selected array(s). If any of the
+       currently selected element(s) is not an array, abort processing and
+       return an error. If the index does not exist in a selected array, remove
+       that array from the selection.
+  3. If the set of elements currently selected is empty, abort processing and
+     return an error.
+
+The result of the processing is the set of elements to which the respective
+claim metadata applies.
+
+Note: The `path` property MUST point to the respective claim as if all
+selectively disclosable claims were disclosed to a Verifier. That means that a
+consuming application which does not have access to all disclosures may not be
+able to identify the claim which is being addressed.
+
+## Claim Display Metadata {#claim-display-metadata}
+
+The `display` property is an array containing display information for the
+claim. The array MUST contain an object for each language that is supported by
+the type. The consuming application MUST use the language tag it considers most
+appropriate for the user.
+
+The objects in the array MUST have the following properties:
+
+- `lang`: A language tag as defined in Section 2 of [@!RFC5646]. This property is REQUIRED.
+- `label`: A human-readable label for the claim, intended for end users. This
+  property is OPTIONAL.
+- `description`: A human-readable description for the claim, intended for end
+  users. This property is OPTIONAL.
+
+## Claim Verification Metadata {#claim-verification-metadata}
+
+The `verification` property is a string indicating how the claim was verified.
+The following values are defined:
+
+- `self-attested`: The claim's value was self-attested by the End-User towards
+  the Issuer. The Issuer did not verify the claim. For example, in a diploma,
+  the residential address of the student may be self-attested.
+- `verified`: The claim's value was verified by the Issuer. The Issuer may have
+  used a third party to verify the claim. For example, in a diploma, the birth
+  date of the student may have been verified by the university using the
+  student's passport.
+- `authoritative`: The Issuer claims to be the authority to make a statement
+  about the claim's value. For example, in a diploma, the degree earned by the
+  student may be authoritative if the Issuer is the university that issued the
+  degree.
```

Comment:
This entire section seems to assume that the Issuer is the same entity that produces/owns/controls/etc the type metadata. Which isn't always or even often the case.

Reply:
As agreed on the call, I'll remove that feature for now

Reply:
As said on the call, thanks. Perhaps put an issue in, if you want to keep track of it? Unless you feel #145 or other is enough.